<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Parametry losowo</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Ile param. drogi:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinIleParamDr">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Ile odcinkow drogi:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinIleOdc">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>999</number>
               </property>
               <property name="value">
                <number>10</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Ile param. pojazdu:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinIleParPoj">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>999</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="buttonLosuj">
             <property name="text">
              <string>Losuj</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Parametry z pliku</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="pushWszystko">
             <property name="text">
              <string>Wczytaj wszystko</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushNazwy">
             <property name="text">
              <string>Wczytaj nazwy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushTrasa">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Wczytaj trasę</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushOgraniczenia">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Wczytaj ograniczenia</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushZaleznosci">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Wczytaj zależności</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Trasa:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tabelkaTrasa">
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Ograniczenia dla samochodu:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tabelkaOgraniczenia"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Ustawienia symulacji</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Wielkość populacji</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinWielkoscPopulacji">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="value">
              <number>20</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Ilość kroków symulacji</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinIloscKrokow">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10000000</number>
             </property>
             <property name="value">
              <number>10000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Algorytm selekcji</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboSelekcja">
             <item>
              <property name="text">
               <string>Metoda ruletki</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Metoda rankingowa</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Metoda turniejowa</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Metoda liniowa wg indeksów</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Prawd. wyboru:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spinPrawd">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Prawd. mutacji</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spinMutacja">
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.050000000000000</double>
             </property>
             <property name="value">
              <double>0.250000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Krzyżowanie</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboKrzyzowanie">
             <item>
              <property name="text">
               <string>Jednopunktowe</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Wielopunktowe</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Prawd. krzyżowania</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spinKrzyzowanie">
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.050000000000000</double>
             </property>
             <property name="value">
              <double>0.250000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkElita">
           <property name="text">
            <string>Wybór elity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkNajlepszy">
           <property name="text">
            <string>Zapamiętaj najlepszego</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="title">
         <string>Najlepszy samochód w aktualnej populacji</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignJustify|Qt::AlignTop</set>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QTableWidget" name="tabelkaNajlepszy"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Ocena</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="najlepszyOcena"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Najlepszy samochód od początku symulacji</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QTableWidget" name="tabelkaNajlepszyGlobalnie"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Ocena</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="najlepszyOcenaGlobalnie"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="buttonStart">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonStop">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonKrok">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Krok</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonResetuj">
        <property name="text">
         <string>Resetuj</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonWyniki">
        <property name="text">
         <string>Wyniki</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushEksport">
        <property name="text">
         <string>Eksport wyników</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressKroki">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
        <property name="format">
         <string>%v/%m</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
